integration/kafka/pom.xml

```diff
@@ -0,0 +1,88 @@
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.alluxio</groupId>
+    <artifactId>alluxio-integration</artifactId>
+    <version>1.1.0-SNAPSHOT</version>
```

Comment:
You can pull master into your branch. Given your change is quite isolated, there shouldn't be much conflict.